.gitignore

```diff
@@ -20,3 +20,6 @@
 # Ignore Byebug command history file.
 .byebug_history
 coverage/
+
+# Ignore vendor/cache/ files.
+/vendor/cache
```

Comment:
I am curious why you are updating the gitignore in this PR? Should this belong elsewhere, or it's own separate PR?